RoaringBitmap/src/main/java/org/roaringbitmap/insights/NaiveWriterRecommender.java

```diff
@@ -78,7 +78,7 @@ private static void containerCountRecommendations(BitmapStatistics basedOn, Stri
       .append(averageContainersCount)
       .append("), because on average each bitmap has ")
       .append(averageContainersCount)
-      .append(" containers.\n");
+        .append(" containers.\n");
```

Comment:
Not sure why, but this file failed `checkstyleMain` after my changes. So I fixed the indentation as checkstyle required.